entrypoint.sh

```diff
@@ -9,8 +9,12 @@ if [ "$DABUILD_DEBUG" = "true" ]; then
 fi
 
 ## generate signing keys on first run
-if [ ! -r "$HOME/.abuild/abuild.conf" ]; then
-  abuild-keygen -n -i -a
+if [ ! -r "$HOME"/.abuild/dabuilder.rsa ]; then
```

Comment:
What guarantee do we have that we should be using the key with that name?

Reply:
The guarantee is in next 3 lines: L13-L15

Reply:
Well, if you don't care about breaking indexes, I don't understand why you have the `if` at all, just always generate a new key every time.

Reply:
On first run I check if the key "$HOME"/.abuild/dabuilder.rsa is already present, if not - I generate it and use it.

Reply:
Yes, I got that, but nothing guarantees my `abuild.conf` was using your key. I have a key in the volume already and used it to sign my built packages and indexes.

Your change to a different key means you are creating at least one index file that my package consuming systems won't trust now.

Reply:
I feel I have a very limited knowledge about workflow intended to be implemented. That's why I can not pay due attention to key management and offer superficial ideas. I would better stick to testing on `macOS` and providing feedback.